sass/layout/_webhooks.scss

```diff
@@ -196,6 +196,11 @@
                 width: 100%;
                 word-wrap: break-word;
             }
+
+            button {
```

Comment:
Can we add this by `.file-preview__button` class? If we change the html around or if we have another button for some reason we will be inheriting this css.